### ISSUE TITLE
Improve error message dialogs

### ISSUE THREAD

**Issue by lipis** [excalidraw/excalidraw#2799]
Not sure what to do.. but this is a bit _ugly_ :)

<img width="576" alt="Screenshot 2021-01-16 at 8 53 30 PM" src="https://user-images.githubusercontent.com/125676/104820354-e64cb180-583c-11eb-9366-e9cef5b0f182.png">

**Comment by inwardmovement** [comment]
Just thinking

![image](https://user-images.githubusercontent.com/9438102/104855104-237f7500-590b-11eb-9c18-9a65584de0bc.png)

**Comment by lipis** [comment]
> Just thinking
> 

Nah.. I wasn't talking about the actual copy of the error.. but in general how it looks like. I wouldn't add icons though..
